content/en/Getting started/sign-in.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Sign In to Cobalt"
+linkTitle: "Sign In"
+weight: 9
+description: >
+  Start the pentest process. Sign in to the Cobalt app.
+---
+
+{{% pageinfo %}}
+Learn about your first steps with Cobalt after receiving a welcome email.
+{{% /pageinfo %}}
+
+## Set Up Your Account
+
+Once you've received a welcome email from Cobalt, do the following:
+
+1. Select **Sign In** in the email.
+1. Create a password. Follow the password complexity requirements on the screen.
+
+Once you've confirmed your email address and created a password, your Cobalt account is fully set up.
+
+## Sign-in Methods
+
+Depending on the configurations of your organization, you can sign in to Cobalt in the following ways:
+
+- Through [SAML single sign-on](#saml-sso), if configured. Go to your identity provider system to sign in to Cobalt.
+  - If your organization has [enforced SAML](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#enforce-saml-sso), authentication from the Cobalt {{% sign-in %}} page is not possible.
+- From the Cobalt {{% sign-in %}} page, with:
+  - Your email address and password.
+  - Your Google account that you used to set up your profile with Cobalt.<br><br>
```

Comment:
"used to set up your profile": wondering if there's a way to change this to something like the Google account that you were invited to Cobalt with? Users can't really select what email they use, because they can't register, but need to be invited.

Reply:
Changed to "Your Google account with which you were invited to Cobalt."